authors/apps/authentication/urls.py

```diff
@@ -4,6 +4,8 @@
     LoginAPIView, RegistrationAPIView, UserRetrieveUpdateAPIView
 )
 
+app_name = 'authentication'
+
 urlpatterns = [
```

Comment:
We should use the 'path' function to set the various URLs on the application so as to follow the new Django standards

Reply:
I think that change was already made or is being handled on this [branch](https://github.com/andela/ah-centauri-backend/tree/ch-upgrade-dependencies-164829189)

@Sm0k3s